### ISSUE TITLE
path to xdebug.so wrong for debian

### ISSUE THREAD

**Issue by it-ony** [jippi/puppet-php#102]
Including `include php::extension::xdebug` leads on my system to write the following xdebug.ini.

```
cat /etc/php5/mods-available/xdebug.ini
zend_extension=/usr/lib/php5/20121212/xdebug.so
```

The module `/usr/lib/php5/20121212/xdebug.so` cannot be found. I checked the path and found out that the file is stored under `/usr/lib/php5/20100525/xdebug.so`.

My system is a debian, installed within vagrant from this base box http://vagrant-boxes.cargomedia.ch/virtualbox/debian-7-amd64-default.box

```
name -a
Linux gamba 3.2.0-4-amd64 #1 SMP Debian 3.2.51-1 x86_64 GNU/Linux
```

**Comment by schkovich** [comment]
@igalic, I would say that this is fixed by #123

**Comment by igalic** [comment]
okay cool.
